docs/operation.md

```diff
@@ -0,0 +1,33 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# User memory configuration of containerPool
+## Change user memory to all invokers via controller. e.g.
+```
+curl -u ${username}:${password} -X POST http://${controllerAddress}:${controllerPort}/config/memory -d '1024 MB'
+```
+Note: you can add `?limit` to specify target invokers, e.g. specify invoker0 and invoker1
+```
+curl -u ${username}:${password} -X POST http://${controllerAddress}:${controllerPort}/config/memory?limit=0:1 -d '1024 MB'
```

Comment:
I feel this API should be based on JSON and have a better protocol.
For example, the name of the invoker may not be simple like just `invoker0`.

Reply:
Currently, i changed it to like below
```
curl  -u admin:admin -X POST http://xxx.xxx.xxx.xxx:10001/config/memory  -d '
[{"invoker":0,"memory": "20480 MB"}, {"invoker":1,"memory": "10240 MB"}]
'
```